src/views/Home/components/CLWCreditsDaysLeft/index.jsx

```diff
@@ -17,6 +25,8 @@ const CLWCreditsDaysLeft = () => {
   const [loading, setLoading] = useState(true);
   const [currSessionDescription, setCurrSessionDescription] = useState('');
 
+  const getColor = (cssVar) => getComputedStyle(document.documentElement).getPropertyValue(cssVar);
```

Comment:
We probably don't want to commit this file as is, since this is not an acceptable final solution. Please undo these changes for now until we find a working solution for chart.js

Reply:
Fair enough, I'll undo those for now :+1: